key/key_test.go

```diff
@@ -0,0 +1,25 @@
+package key
+
+import (
+	"encoding/hex"
+	"testing"
+
+	"github.com/CodeChain-io/codechain-rpc-go/crypto"
+)
+
+func TestEcdsa(t *testing.T) {
```

Comment:
`crypto/ecdsa_test.go` has same function name with this.
And I think `ecdsa_test.go` should be modified not this.
So I'll approve this pr.